File: CursoJava/Exer/EstruturaCondicional/src/Exer03.java
/*
Leia 2 valores inteiros (A e B). Após, o programa deve mostrar uma mensagem "Sao Multiplos" ou "Nao sao
Multiplos", indicando se os valores lidos são múltiplos entre si. Atenção: os números devem poder ser digitados em
ordem crescente ou decrescente.
 */

import java.util.Scanner;

public class Exer03 {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Digite dois números inteiros: ");
        int A = sc.nextInt();
        int B = sc.nextInt();

        if (A % B == 0 || B % A == 0) {
            System.out.println("São Múltiplos");
        } else {
            System.out.println("Não são Múltiplos");
        }

        sc.close();
    }
}

File: CursoJamiltonDamasceno/CursoWebCompleto/exer/estruturaCondicional/exer03.js
/*
3. Leia 2 valores inteiros (A e B). Após, o programa deve mostrar uma mensagem
"Sao Multiplos" ou "Nao sao Multiplos", indicando se os valores lidos são múltiplos entre si.
Atenção: os números devem poder ser digitados em ordem crescente ou decrescente.
 */

const a = 10;
const b = 15;

if (a % b === 0 || b % a === 0) {
    console.log("São múltiplos");
} else {
    console.log("Não são múltiplos");
}